_rules/role-attribute-valid-value-674b10.md

```diff
@@ -60,23 +65,23 @@ Older browsers do not support more than one token in the value for a role attrib
 
 #### Passed Example 1
 
-Element with valid `role` value.
+This [`role` attribute][role attribute] contains one token, and this token is a valid [WAI-ARIA role][].
 
 ```html
 <input type="text" role="textbox" />
 ```
 
 #### Passed Example 2
 
-Element with multiple valid `role` values.
+This [`role` attribute][role attribute] contains two tokens, and these tokens are both valid [WAI-ARIA roles][wai-aria role].
 
 ```html
 <span role="button link"></span>
 ```
 
 #### Passed Example 3
 
-Element with at least one valid `role` value.
+This [`role` attribute][role attribute] contains two tokens, and one of these tokens is a valid [WAI-ARIA role][].
 
 ```html
 <img role="img xyz" src="/test-assets/shared/w3c-logo.png" alt="W3C logo" />
```

Comment:
This example could be upgraded a bit by using another value than "xyz" that looks like a "plausible typo".
I preferred not touching the examples to keep implementation data.

Reply:
I like that idea. I think we should just make that change.